test/WebJobs.Script.Tests/EndToEndTestFixture.cs

```diff
@@ -33,6 +34,11 @@ protected EndToEndTestFixture(string rootPath)
             CreateTestStorageEntities();
             TraceWriter = new TestTraceWriter(TraceLevel.Verbose);
 
+            // clean up entities that might be left from previous run
+            // this is before starting the host so any trigger that starts with the host 
+            // is not running yet
+            CleanupEntities().Wait();
```

Comment:
Recommend moving this to before `CreateTestStorageEntities` above. Also rename `CreateTestStorageEntities` to `CreateTestEntities` and rename your function to `CleanupTestEntities`.